mappings/net/minecraft/network/message/ChatDecorator.mapping

```diff
@@ -0,0 +1,56 @@
+CLASS net/minecraft/class_7492 net/minecraft/network/message/ChatDecorator
```

Comment:
Seems this doesnt merge nicely now it was renamed. I tried rebasing (using the bot) but that doesnt seem to have helped.

Reply:
@modmuss50 That's okay, I can fix it